app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -28,6 +28,42 @@ def parent_cluster
   alias_method :owning_cluster, :parent_cluster
   alias_method :ems_cluster, :parent_cluster
 
+  def self.create_from_event(event)
```

Comment:
Not only save_vm_inventory is responsible for creating relationships so this makes it unusable in this context.